packages/zowe-explorer/package.json

```diff
@@ -2092,36 +2094,54 @@
     "vscode": "^1.79.0"
   },
   "devDependencies": {
+    "@cucumber/cucumber": "9.5.1",
     "@napi-rs/cli": "^2.16.1",
     "@types/chai": "^4.2.6",
     "@types/chai-as-promised": "^7.1.0",
     "@types/expect": "^1.20.3",
     "@types/fs-extra": "^7.0.0",
     "@types/promise-queue": "^2.2.0",
+    "@types/sinon": "^17.0.3",
+    "@wdio/browser-runner": "^8.37.0",
```

Comment:
Are we using the browser runner, the local runner, or both? I assumed only one could be used at a time and am curious if both dependencies are needed but am probably assuming wrong 😋

Reply:
We are currently using the `local` runner, but we can also leverage the `browser` runner to test in web environments. I felt that might be useful in case we wanted to expand support for web instances of VS Code, so I left the dependency here from the research phase.